src/main/java/net/dries007/tfc/common/blocks/devices/PowderkegBlock.java

```diff
@@ -121,4 +121,12 @@ public boolean canDropFromExplosion(BlockState state, BlockGetter level, BlockPo
     {
         return false;
     }
+
+    @Override
+    public void neighborChanged(BlockState state, Level level, BlockPos pos, Block blockIn, BlockPos fromPos, boolean isMoving) {
```

Comment:
bracket is in the wrong spot

Reply:
Ugh. Thought I had the codestyle loaded. :/